tests/integration/solidity_contracts/PlainOpcodes/PlainOpcodes.t.sol

```diff
@@ -48,4 +49,53 @@ contract PlainOpcodesTest is Test {
         bytes memory bytecode = plainOpcodes.opcodeExtCodeCopy(0, counterSize);
         assertEq0(bytecode, expectedResult);
     }
+
+    function testShouldAlwaysRevert() public {
+        vm.expectRevert(bytes("This always reverts"));               
+        revert("This always reverts");
+    }
+
+    function testCallingContextShouldPropogateRevertFromSubContext() public {
+        Parent parent = new Parent();
+        vm.expectRevert(bytes("FAIL"));                                    
+        parent.triggerRevert();
+
+    }    
+    
+    function testShouldRevertViaCall() public {
+        Parent parent = new Parent();
+        // child contract is created in the same execution context as a revert       
+        (bool success, ) = address(parent).call(abi.encodeWithSignature("triggerRevert()"));
+
+        require(!success, "trigger revert... should revert");                
```

Comment:
could yse `assert` instead of adding unnecessary revert reason `"trigger revert... should revert"`